htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -341,6 +342,9 @@ public function create(User $user, $notrigger = false)
 		}
 		$sql .= " AND numero_compte = '".$this->db->escape($this->numero_compte)."'";
 		$sql .= " AND label_operation = '".$this->db->escape($this->label_operation)."'";
+		$sql .= " AND montant = '".((float) $this->montant)."'";
```

Comment:
If you introduce this lines, it means you want to remove the protection.
Without the protection, it will be possible to have duplicate line with same
doc_type, doc id, entity, account number and account label.
I don't see any use cases when we need to have such a duplicate. Can you provide an example of value and use cas that show you need to be able to insert such duplicate ?

Reply:
@eldy You were right